lstchain/scripts/lstchain_data_muon_analysis_dl1.py

```diff
@@ -99,7 +100,10 @@ def main():
             mirror_area = telescope_description['mirror_area'].values * pow(u.m,2)
             
             for full_image, event_id, dragon_time in zip(images, parameters['event_id'], parameters['dragon_time']):
-                image = full_image*(~bad_pixels)
+                if args.calib_file is not None:
+                    image = full_image*(~bad_pixels)
```

Comment:
`image[bad_pixels] = 0`

Reply:
> `image[bad_pixels] = 0`

It is not the same... and I would have to take care later that the pixel coordinates' arrays are also masked, passing the bad pixels to other functions. I prefer not to take the risk now.